src/main/scala/server/Config.scala

```diff
@@ -1,11 +1,13 @@
 package server
 
 import com.typesafe.config.ConfigFactory
+import scala.util.Properties.envOrElse
 
 object Config {
   private val conf = ConfigFactory.load
-  val host = conf.getString("cdserver.host")
-  val port = conf.getInt("cdserver.port")
+
+  val host = envOrElse("HOST", "localhost")
```

Comment:
@ben-mulhern this is where I need a bit of advice (not a scala dev) - we need these to come from environment variables so we can change them when deploying to non-local infrastructure. I found this library and it works, but I couldn't get it to work within application.conf (I'm guessing that the file needed to be a `.scala` to use it?)

Oof. Don't know Scala, so looking to you for some advice on whether what i've done is okay. Basically all I did was take those two settings (host and port) out of the application.conf file and put them in-line where you'd imported them, and used the envOrElse library to take them from env vars OR use the defaults you used to have.